src/lib/config/defaultConfig.js

```diff
@@ -261,6 +261,11 @@ export const RECORDS_CONFIG = {
           field: 'stock.mediums',
           aggName: 'medium',
         },
+        {
+          title: 'Restricted',
```

Comment:
are you sure this will not cause that the filter will appear on the frontsite search ? In general we wouldn't want that, since the frontsite search should not display restricted documents by default

Reply:
Good guess... to be checked ;) Is this solving an issue? I don't see it mentioned in the PR.

Reply:
My bad, I didn't add the issue on the PR, sorry ... The filter is added to the backoffice as requested on the issue now linked #28 . The `restricted` filter is only present in the backend since in the frontsite we use the [literature](https://github.com/inveniosoftware/react-invenio-app-ils/blob/master/src/lib/config/defaultConfig.js#L590) filters instead the [documents](https://github.com/inveniosoftware/react-invenio-app-ils/blob/master/src/lib/config/defaultConfig.js#L223) filters (these are the ones used in the backoffice)